ls1intum/Artemis
# `General`: Fix an issue when deleting exercises with existing tutor participations

### Checklist
#### General
#### Server

### Motivation and Context
Deleting a text or modeling exercise is not possible if this exercise has an example submission with tutor participations. The following internal server error gets shown: `failed to lazily initialize a collection of role: de.tum.in.www1.artemis.domain.participation.TutorParticipation.trainedExampleSubmissions, could not initialize proxy - no Session`

### Description
I used the corresponding repository to first delete all tutor participations before deleting the exercise.

### Steps for Testing
1. Create a new text or modeling exercise
2. Create an example submission for this exercise
3. As a Tutor, open the Assessment Dashboard of the exercise and accept the grading instruction. Then look at the example submission and save your progress
4. As an Instructor, delete the exercise. Check that no error is thrown and the deletion works.

### Review Progress

#### Code Review
#### Manual Tests
